rust-toolchain

```diff
@@ -1,3 +1 @@
-[toolchain]
-channel = "stable"
-components = ["clippy", "rustfmt"]
+1.47
```

Comment:
Keep the new format. Just change the `channel` line.

Reply:
Fixed